services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -85,45 +98,46 @@ const normalizeAccount = account => {
 	return parseToJSONCompatObj(account);
 };
 
-const getAccounts = async params => {
+const getAccountsFromCore = async (params) => {
 	const accounts = {
 		data: [],
 		meta: {},
 	};
+	const response = await coreApi.getAccounts(params);
+	if (response.data) accounts.data = response.data.map(account => normalizeAccount(account));
+	if (response.meta) accounts.meta = response.meta;
+	return accounts;
+};
 
-	const requestParams = {
-		limit: params.limit,
-		offset: params.offset,
-		sort: params.sort,
-		username: params.username,
-	};
+const getAccounts = async params => {
+	const accountsDB = await knex('accounts');
 
 	if (params.address && typeof params.address === 'string') {
-		const parsedAddress = parseAddress(params.address);
-		if (!(await confirmAddress(parsedAddress))) return {};
-		requestParams.address = parsedAddress;
+		if (!(await confirmAddress(params.address))) return {};
 	}
 	if (params.publicKey && typeof params.publicKey === 'string') {
 		if (!validatePublicKey(params.publicKey) || !(await confirmPublicKey(params.publicKey))) {
 			return {};
 		}
-		requestParams.publicKey = params.publicKey;
-	}
-	if (params.secondPublicKey && typeof params.secondPublicKey === 'string') {
-		if (!validatePublicKey(params.secondPublicKey)
-			|| !(await confirmSecondPublicKey(params.secondPublicKey))
-		) {
-			return {};
-		}
-		requestParams.secondPublicKey = params.secondPublicKey;
 	}
-
-	const response = await coreApi.getAccounts(requestParams);
-	if (response.data) accounts.data = response.data.map(account => normalizeAccount(account));
-	if (response.meta) accounts.meta = response.meta;
-
-	// accounts.data = await resolveAccountsInfo(accounts.data);
-
+	if (params.isDelegate) params.isDelegate = validateBoolean(params.isDelegate);
```

Comment:
With `params.isDelegate` being mapped `boolean` at the gateway, is `validateBoolean` necessary? 
`params.isDelegate` should not be containing any number or string. Is gateway not parsing the boolean request parameters properly?

```suggestion
```